less/type.less

```diff
@@ -238,6 +238,67 @@ ol {
   }
 }
 
+ul.tw-list-styled {
```

Comment:
Would it be possible to rename the class 'circle-list' to draw comparison with our regular circle classes?